docs/tools/pow-vs-pos-vs-dpos.md

```diff
@@ -0,0 +1,63 @@
+id: pow-vs-pos-vs-dpos
+title: PoW vs PoS vs DPoS
+sidebar_label: PoW vs PoS vs DPoS
+---
+
+It is necessary to know the consensus mechanisms used by the different blockchain networks to maintain security and integrity within the network, as well as the benefits and the networks that still conform it.
+
+## Proof of Work (PoW)
+
+<figure class="video_container">
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/3EUAcxhuoU4" frameborder="0" allowfullscreen="true"> </iframe>
+</figure>
+
+It is the first consensus mechanism presented by Satoshi Nakamoto in the Bitcoin whitepaper in 2008, in this consensus a group of transactions called "Mempool" are verified by the "miners", who solve a complex mathematical problem (which increases in complexity the more blocks are validated) so specialized computer equipment is required in this case of "mining" and in turn large amounts of energy.
+
+The first miner who finds the solution to the problem and this is tested by the other miners obtains the right to add the "Mempool" to the blockchain and obtain a "Block reward" composed of cryptocurrencies generated by the transaction fees.
+
+Due to the high requirements to be able to "mine", mining groups or "Pool mining" have been formed in order to find the solution to the mathematical problem with the use of resources, thus having better chances of finding the solution first than individual miners with greater resources and the reward obtained is distributed among the members of the pool.
+
+Currently the networks that use this consensus mechanism are:
+
+- Bitcoin.
+- Ethereum (In the future it is possible that it will change to PoS).
+- Monero.
+- Among others.
+
+For more information you can visit the following [link](https://guide.eoscostarica.io/docs/tools/glossary/#stake).
+
+## Proof of Stake (PoS)
+
+<figure class="video_container">
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/psKDXvXdr7k" frameborder="0" allowfullscreen="true"> </iframe>
+</figure>
+
+This consensus mechanism was born in 2011 with the purpose of eliminating those limitations present in its predecessor proof of work, having instead of "miners" that require resources for the verification of transactions such as specialized equipment and large amounts of energy to "validators" who are owners of coins and put as collateral ["Stake"](https://guide.eoscostarica.io/docs/tools/glossary/#stake) a certain amount of coins, being chosen randomly the validators responsible for "validating" or "mining" in case there is a corrupt validator will lose the stake and will not be able to be a validator in the future.
+
+For the validation of transactions, 2/3 (this figure may vary depending on the network) of the total number of validators are required to approve the transaction block. The following aspects are taken into consideration for the choice of validators:
+
+- Amount of coins (the higher the amount, the higher the probability of being chosen).
```

Comment:
Amount of coins

replace with amount of tokens staked

Reply:
Fixed